http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20170811_label.xml

```diff
@@ -12,10 +12,5 @@
                 <constraints nullable="false"/>
             </column>
         </createTable>
-        <createIndex indexName="FK_CLUSTER_ID" tableName="LABEL">
-            <column name="clusterId"/>
-        </createIndex>
-        <addForeignKeyConstraint baseColumnNames="clusterId" baseTableName="LABEL" constraintName="FK_CLUSTER_ID" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="CLUSTER"/>
-        <addUniqueConstraint columnNames="clusterId, key" constraintName="IDX_LABEL_UNIQUE" tableName="LABEL"/>
```

Comment:
is this FK not useful anymore or just not supported?

Reply:
No idea, this is me just testing a theory that this might be the problematic changeset. Definitely a throwaway commit not ready for review 😓